io.itch.drwhut.TabletopClub.json

```diff
@@ -0,0 +1,57 @@
+{
+    "app-id": "io.itch.drwhut.TabletopClub",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "22.08",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "tabletop-club",
+    "finish-args": [
+        "--socket=x11",
+        "--socket=fallback-x11",
+        "--socket=wayland",
```

Comment:
does it support wayland?

it's either x11 alone or fallback-x11 with wayland.

Reply:
Yes it does, I'll remove the x11 on it's own.